index.ts

```diff
@@ -605,6 +627,8 @@ function tsdd(suite) {
         context.xit = context.xspecify = context.it.skip = function(title) {
             context.it(title);
         };
+
+        /* istanbul ignore next */
```

Comment:
I believe that we should add a test that actually covers this. Otherwise, this is unused code that should be eliminated.

Reply:
same